src/bool.cr

```diff
@@ -55,4 +55,9 @@ struct Bool
   def to_s(io)
     io << to_s
   end
+
+  # Returns `self`
```

Comment:
Oh, I forgot to enhance comments for clone. Do you think "Returns `self` (Bool is immutable)" and similar comments in other types would make this clearer?